### ISSUE TITLE
Multiple ROI Manager issues from Import DICOM window

### ISSUE THREAD

**Issue by cutright** [cutright/DVH-Analytics#102]
1) If adding physician ROI by right-click:
```
ERROR: Unhandled exception: Traceback (most recent call last):
  File "C:\Users\dcutright\PycharmProjects\DVH-Analytics\dvha\dialogs\roi_map.py", line 425, in add_physician_roi
    AddPhysicianROI(self.parent, self.physician, self.roi_map)
  File "C:\Users\dcutright\PycharmProjects\DVH-Analytics\dvha\dialogs\roi_map.py", line 646, in __init__
    self.run()
  File "C:\Users\dcutright\PycharmProjects\DVH-Analytics\dvha\dialogs\roi_map.py", line 708, in run
    self.action()
  File "C:\Users\dcutright\PycharmProjects\DVH-Analytics\dvha\dialogs\roi_map.py", line 719, in action
    self.roi_map.add_variations(self.physician, self.text_ctrl_physician_roi.GetValue(), self.physician_roi)
  File "C:\Users\dcutright\PycharmProjects\DVH-Analytics\dvha\tools\roi_name_manager.py", line 475, in add_variations
    self.physicians[physician].add_variations(physician_roi, variation)
  File "C:\Users\dcutright\PycharmProjects\DVH-Analytics\dvha\tools\roi_name_manager.py", line 114, in add_variations
    self.rois[physician_roi].add_variations(variations)
  File "C:\Users\dcutright\PycharmProjects\DVH-Analytics\dvha\tools\roi_name_manager.py", line 43, in add_variations
    if clean_name(variation) not in clean_variations and variation.lower() not in {'uncategorized'}:
AttributeError: 'NoneType' object has no attribute 'lower'
```
2) From ROI Manager, Physician ROI drop down list does not update after adding a new Physician ROI
3) Clicking Delete after a Select All button doesn't do anything, but then selecting an ROI manually and then clicking delete, deletes some variations (maybe just those in view? unclear...)

**Comment by cutright** [comment]
RE: Item 3

```
ERROR: Unhandled exception: Traceback (most recent call last):
  File "C:\Users\dcutright\PycharmProjects\DVH-Analytics\dvha\dialogs\roi_map.py", line 402, in delete_variations
    self.roi_map.delete_variations(self.physician, self.physician_roi, self.selected_values)
  File "C:\Users\dcutright\PycharmProjects\DVH-Analytics\dvha\tools\roi_name_manager.py", line 483, in delete_variations
    self.physicians[physician].del_variations(physician_roi, variations)
  File "C:\Users\dcutright\PycharmProjects\DVH-Analytics\dvha\tools\roi_name_manager.py", line 119, in del_variations
    self.rois[physician_roi].del_variation(variations)
  File "C:\Users\dcutright\PycharmProjects\DVH-Analytics\dvha\tools\roi_name_manager.py", line 70, in del_variation
    self.variations.pop(index)
IndexError: pop index out of range
```

**Comment by cutright** [comment]
Unable to reproduce item 2

**Comment by cutright** [comment]
4) Should not be able to edit DEFAULT physician from this window. DEFAULT is used for "institutional" ROIs.

**Comment by cutright** [comment]
Fixed in `v0.8.8`
